test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2454,6 +2454,27 @@ def forward(self, input):
         x = torch.randn(1, 2, 3, requires_grad=True)
         self.run_test(EmptyBranchModel(), x)
 
+    def test_onnx_proto_checker(self):
```

Comment:
Let's increase the variety of test coverage here. We want to ensure that failures are indeed captured, and warnings emitted.
Checking for invalid opset version, invalid operator definition, and SSA format are good examples.
Also do we know if the export is going to fail for models with custom operators?